src/goto-programs/goto_convert.cpp

```diff
@@ -1258,6 +1258,7 @@ void goto_convertt::convert_switch(
 
   goto_programt tmp_cases;
 
+  size_t case_number=1;
```

Comment:
the pretty-printing is for comments, not for code, so these have to be adapted to fit, but this will probably not be too difficult